frontend/src/compositions/IndividualSupermarketCart/IndividualSupermarketCart.tsx

```diff
@@ -43,11 +28,12 @@ export function IndividualSupermarketCart({
           data-testid="supermarket-spend"
           className="text-4xl font-semibold"
         >
-          ${supermarketSpend}
+          ${cartCosts[shopCode]}
         </Text>
       </div>
-      {devInitialCart[shopCode] &&
-        Object.entries(devInitialCart[shopCode]).map(([id, item]) => (
+
+      {supermarketCart &&
+        Object.entries(supermarketCart).map(([id, item]) => (
           <CartProduct cartProductItem={item} key={id} />
```

Comment:
💅 A suggestion purely in case you think it's clearer, go with whatever you prefer. I find `Object.entries` weird to look at but I know that's a me thing 😅 
```suggestion
        Object.values(supermarketCart).map((item) => (
          <CartProduct cartProductItem={item} key={item.id} />
```